### PR TITLE
Module 3 - Transaction model

### PR DESCRIPTION
**Pull request by sinpai**
* Added transaction model and migration for it

### CODE REVIEW

File: app/models/transaction.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+class Transaction < ApplicationRecord
+  belongs_to :user
+
+  validates :sum, :date, presence: true
+  validates :sum, numericality: true
+
+  scope :most_recent, ->(user) { where(user_id: user).order(created_at: :desc).limit(10) }
+  scope :total, ->(user) { sum(:sum).where(user_id: user) }
```
Comment (AleksSenkou) [inline app/models/transaction.rb]: @sinpai you've added two scopes here, but I don't see where they used in the PR. If you are planning to use them later - let's add the later.

Comment (AleksSenkou) [reply app/models/transaction.rb]: @sinpai after all, we have to have tests for those scopes. they can be located in `spec/models/transaction_spec.rb` file

**Review by AleksSenkou — COMMENTED** [review]
Suggestions and Q

File: app/models/transaction.rb (hunk 1 above)
Comment (AleksSenkou) [reply app/models/transaction.rb]: FYI: `spec/models/transaction_spec.rb`

Comment (sinpai) [reply app/models/transaction.rb]: @AleksSenkou 👍 I've removed those scopes from this PR. I will add it in PRs, when it's required to use them (along with tests).

**Review by AleksSenkou — APPROVED** [review]
LGTM 🥇

**Pull request merged**